roofit/roofitcore/src/RooVectorDataStore.cxx

```diff
@@ -1486,3 +1487,33 @@ RooVectorDataStore::RealFullVector* RooVectorDataStore::addRealFull(RooAbsReal*
 
   return _realfStoreList.back() ;
 }
+
+
+/// Trigger a recomputatin of the cached weight sums. Meant for use by RooFit
+/// dataset converter functions such as the NumPy converter functions
+/// implemented as pythonizations. Not meant to be part of the public user
+/// interface, so the interface might change without warning.
+void RooVectorDataStore::recomputeSumWeight() {
+  double const* arr = nullptr;
+  if (_extWgtArray) {
+    arr = _extWgtArray;
+  }
+  if (_wgtVar) {
+    std::string wgtName = _wgtVar->GetName();
+    for(auto const* real : _realStoreList) {
+      if(wgtName == real->_nativeReal->GetName())
+      arr = real->_vec.data();
```

Comment:
Indent or put in one line? Not so nice to have the `true` branch on the same indentation level.

Reply:
Ok, right I forgot the indentation!